armi/reactor/reactors.py

```diff
@@ -156,7 +161,7 @@ def factory(cs, bp, geom: Optional[SystemLayoutInput] = None) -> Reactor:
         coreDesign.construct(cs, bp, r, geom=geom)
         for structure in bp.systemDesigns:
             if structure.name.lower() != "core":
-                structure.construct(cs, bp, r)
+                structure.construct(cs, bp, r, loadAssems=False)
```

Comment:
Can you provide some context on this change for me? I don't immediately see why it was made.

Reply:
`loadAssems` was causing failures in test_rz_reactors.py because there wasn't any `gridDesign` defined for the RZT reactors. But setting `loadAssems=False` was the wrong approach because we do actually want to load the assemblies from the blueprints, if they're there.

I changed the logic in reactorBlueprint.py to skip `loadAssems` if there is no `gridDesign` available, and now the tests pass without this argument.